tests/drivers/build_all/app.overlay

```diff
@@ -0,0 +1,1029 @@
+				label = "LIS2DH";
+				reg = <0x20>;
+				irq-gpios = <&test_gpio 0 0>;
+			};
+			*/
+
+			si7006@21 {
+				compatible = "silabs,si7006";
+				label = "SI7006";
+				reg = <0x21>;
+			};
+
+			lsm9ds0-mfd@22 {
+				compatible = "st,lsm9ds0-mfd";
+				label = "LSM9DS0-MFD";
+				reg = <0x22>;
+				irq-gpios = <&test_gpio 0 0>;
+			};
+
+			isl29035@23 {
+				compatible = "isil,isl29035";
+				label = "ISL29035";
+				reg = <0x23>;
+				int-gpios = <&test_gpio 0 0>;
+			};
+
+			amg88xx@24 {
+				compatible = "panasonic,amg88xx";
+				label = "AMG88XX";
+				reg = <0x24>;
+				int-gpios = <&test_gpio 0 0>;
+			};
+
+			stts751@25 {
+				compatible = "st,stts751";
+				label = "STTS751";
+				reg = <0x25>;
+				drdy-gpios = <&test_gpio 0 0>;
+			};
+
+			hp206c@26 {
+				compatible = "hoperf,hp206c";
+				label = "HP206C";
+				reg = <0x26>;
+			};
+
+			iaqcore@27 {
+				compatible = "ams,iaqcore";
+				label = "IAQCORE";
+				reg = <0x27>;
+			};
+
+			/*
+			lps22hh@28 {
+				compatible = "st,lps22hh";
+				label = "LPS22HH";
+				reg = <0x28>;
+				drdy-gpios = <&test_gpio 0 0>;
+			};
+			*/
+
+			mcp9808@29 {
+				compatible = "microchip,mcp9808";
+				label = "MCP9808";
+				reg = <0x29>;
+				int-gpios = <&test_gpio 0 0>;
+			};
+
+			opt3001@2a {
+				compatible = "ti,opt3001";
+				label = "OPT3001";
+				reg = <0x2a>;
+			};
+
+			lis2dh12@2b {
+				compatible = "st,lis2dh12";
+				label = "LIS2DH12";
+				reg = <0x2b>;
+				irq-gpios = <&test_gpio 0 0>;
+			};
+
+			mpu6050@2c {
+				compatible = "invensense,mpu6050";
+				label = "MPU6050";
+				reg = <0x2c>;
+				int-gpios = <&test_gpio 0 0>;
+			};
+
+			/*
+			lis2mdl@2d {
+				compatible = "st,lis2mdl";
+				label = "LIS2MDL";
+				reg = <0x2d>;
+				irq-gpios = <&test_gpio 0 0>;
+			};
+			*/
+
+			sx9500@2e {
+				compatible = "semtech,sx9500";
+				label = "SX9500";
+				reg = <0x2e>;
+				int-gpios = <&test_gpio 0 0>;
+			};
+
+			ccs811@2f {
+				compatible = "ams,ccs811";
+				label = "CCS811";
+				reg = <0x2f>;
+				wake-gpios = <&test_gpio 0 0>;
+				reset-gpios = <&test_gpio 0 0>;
+				irq-gpios = <&test_gpio 0 0>;
+			};
+
+			bme280@30 {
+				compatible = "bosch,bme280";
+				label = "BME280";
+				reg = <0x30>;
+			};
+
+			lsm9ds0-gyro@31 {
+				compatible = "st,lsm9ds0-gyro";
+				label = "LSM9DS0-GYRO";
+				reg = <0x31>;
+				irq-gpios = <&test_gpio 0 0>;
+			};
+
+			tmp112@32 {
+				compatible = "ti,tmp112";
+				label = "TMP112";
+				reg = <0x32>;
+			};
+
+			bmg160@33 {
+				compatible = "bosch,bmg160";
+				label = "BMG160";
+				reg = <0x33>;
+				int-gpios = <&test_gpio 0 0>;
+			};
+
+			sht3xd@34 {
+				compatible = "sensirion,sht3xd";
+				label = "SHT3XD";
+				reg = <0x34>;
+				alert-gpios = <&test_gpio 0 0>;
+			};
+
+			max30101@35 {
+				compatible = "max,max30101";
+				label = "MAX30101";
+				reg = <0x35>;
+			};
+
+			si7060@36 {
+				compatible = "silabs,si7060";
+				label = "SI7060";
+				reg = <0x36>;
+			};
+
+			/*
+			lis2ds12@37 {
```

Comment:
I read the description saying these are commented out to match how the test works today, but I don't get why that is necessary with the move to DT. Disabled nodes shouldn't have any effect, right? What am I missing?

Reply:
Nothing, wasn't thinking to just set 'status = disabled'.  Will make that change as I like that solution better.